### PR TITLE
feat: add woomer

### PR DESCRIPTION
**Pull request by diogomdp**

### CODE REVIEW

**Comment by coffeeispower** [comment]
I developed this application, it allows you to zoom your screen in and out, and also has a spotlight when you hold <kbd>CTRL</kbd>.
woomer demo

File: README.md [hunk 1]
```diff
@@ -185,6 +185,9 @@ These technically aren't hyprland plugins, but extend hyprland functionality usi
 
 - [swappy](https://github.com/jtheoof/swappy) ![c][c] (Annotation gui made in gtk)
 
+#### Zoomer application
```
Comment (yavko) [inline README.md:188]: This section name isn't the best, something like `Zooming utilities` would be better.

File: README.md [hunk 2]
```diff
@@ -185,6 +185,9 @@ These technically aren't hyprland plugins, but extend hyprland functionality usi
...
+- [woomer](https://github.com/coffeeispower/woomer) ![rust][rs] (Zoomer application for Wayland inspired by tsoding's boomer)
```
Comment (yavko) [inline README.md:189]: For the description, plz make it be `Zooming utility ....`

**Review by yavko — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (BackHDLP) [reply README.md:189]: reads like "Zoomer application — for Zoomers, by Zoomers" lol

Comment (yavko) [reply README.md:189]: fr fr

Comment (coffeeispower) [reply README.md:189]: That's the joke...

Comment (diogomdp) [reply README.md]: Fixed!

**Review by yavko — APPROVED** [review]

**Pull request merged**